cmd/kube-controller-manager/app/apps.go

```diff
@@ -22,9 +22,23 @@ package app
 
 import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/kubernetes/pkg/controller/replicaset"
 	"k8s.io/kubernetes/pkg/controller/statefulset"
 )
 
+func startReplicaSetController(ctx ControllerContext) (bool, error) {
+	if !ctx.AvailableResources[schema.GroupVersionResource{Group: "extensions", Version: "v1beta1", Resource: "replicasets"}] {
```

Comment:
`extensions/v1beta1 replicasets`? Aren't we moving to a new version? We've already bumped to `apps/v1beta2`.